apps/judicial-system/web/src/routes/Shared/SignedVerdictOverview/SignedVerdictOverview.tsx

```diff
@@ -518,482 +524,510 @@ export const SignedVerdictOverview: React.FC = () => {
   }
 
   return (
-    <PageLayout
-      workingCase={workingCase}
-      activeSection={Sections.CASE_CLOSED}
-      isLoading={isLoadingWorkingCase}
-      notFound={caseNotFound}
-    >
-      <PageHeader
-        title={formatMessage(titles.shared.closedCaseOverview, {
-          courtCaseNumber: workingCase.courtCaseNumber,
-        })}
-      />
-      <FormContentContainer>
-        <Box marginBottom={5}>
-          <Box marginBottom={3}>
-            <Button
-              variant="text"
-              preTextIcon="arrowBack"
-              onClick={() => router.push(constants.CASES_ROUTE)}
-            >
-              Til baka
-            </Button>
-          </Box>
-          <Box display="flex" justifyContent="spaceBetween" marginBottom={3}>
-            <Box>
-              <Box marginBottom={1}>
-                <Text as="h1" variant="h1">
-                  {titleForCase(formatMessage, workingCase)}
-                </Text>
-              </Box>
+    <>
+      {workingCase.courtEndTime &&
```

Comment:
Shouldn't the possibility to appeal also depend no the prosecutor decision made in court? If `prosecutorAppealDecision` was either `Appeal` or `Accept` then I would think the prosecutor should not be able to appeal here.